manager/src/static/js/sscape.js

```diff
@@ -1745,7 +1753,7 @@ $(document).ready(function () {
         $("#save-rois, #save-trips").on("click", function (event) {
           var tripwire_values = getRoiValues(
             "form-control tripwire-title",
-            "tripwire",
+            "tripwire"
```

Comment:
Please keep the coding convention used in js codebase so far.

The rationale behind:
- Consistency: JS codebase is already using this convention, let's keep it consistent. Also, many style guides and tools (Prettier, ESLint) recommend trailing commas for multi-line objects and arrays.
- Cleaner diffs:
When adding or removing items, only the changed line is modified in version control, making diffs easier to read.
- Easier editing:
You can add new elements without worrying about whether to add or remove a comma from the previous line.


```suggestion
            "tripwire",
```